apps/aeutils/priv/aeternity_config_schema.json

```diff
@@ -370,6 +370,20 @@
                             "minimum" : 0
                         }
                     }
+                },
+                "protocol_beneficiaries_enabled" : {
+                    "description" :
+                    "If true, the node will split rewards and send part to protocol_beneficiaries",
+                    "type" : "boolean",
+                    "default" : true
+                },
+                "protocol_beneficiaries" : {
+                    "type" : "array",
+                    "items" : {
+                        "description" : "Public keys belonging to protocol maintainers with reward shares (100 is 10%).",
+                        "pattern": "^ak_[1-9A-HJ-NP-Za-km-z]*:[0-9]+$"
```

Comment:
Can share of a beneficiary be configured to be zero?

Reply:
The regexp allows it, but it's harmless. (same reason as above - no balance update happens).